### ISSUE TITLE
[MU4 Issue] First entered chord doesn't have a sound during input if added to any staff other than piano

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#13206]
**To Reproduce**
1. Create score with non-piano staff (Flute or Violin)
2. Add some notes
3. Select first note
4. Ctr/Cmd+K to enable chord input
5. Press C and Space - **no sound**

https://user-images.githubusercontent.com/90187801/188693268-3e794b36-b847-474e-932a-a61ec3319e5a.mp4

The issue doesn't occur if to enter chord slowly

**Platform information**
macOS, Windows, Linux

**Additional context**
Doesn't occur in **MS3**

**Comment by zacjansheski** [comment]
@DmitryArefiev does this still happen?  MuseSounds takes a little time to fire up for me, but I haven't experienced the issue with the first chord.

**Comment by DmitryArefiev** [comment]
@zacjansheski Yeah. It occurs with **MS Basic only** (can't repro with MuseSounds). Will update summary.

Also, it's more noticeable on Windows

https://github.com/musescore/MuseScore/assets/90187801/aee9c01c-24e3-4d0c-aab3-377e4e59a82d
